tests/pytests/unit/modules/test_disk.py

```diff
@@ -0,0 +1,322 @@
+"""
+    :codeauthor: Jayesh Kariya <jayeshk@saltstack.com>
+"""
+import sys
+
+import pytest
+
+import salt.modules.disk as disk
+import salt.utils.path
+import salt.utils.platform
+from tests.support.mock import MagicMock, patch
+from tests.support.unit import skipIf
+
+STUB_DISK_USAGE = {
```

Comment:
We want to avoid using mutable types at the module/global level for tests. One solution is to use a pytest fixture. This is docuemnted here: https://docs.saltproject.io/en/master/topics/development/tests/index.html#module-global-level-variables

This also applies to STUB_DISK_INODEUSAGE, STUB_DISK_PERCENT and STUB_DISK_BLKID

Reply:
@Ch3LL , this was file ported over from the "old" tests as a nicety. Is it a hard blocker, or can we possibly merge as-is?

Reply:
Its a hard blocker as these mutable objects can end up causing issues that are hard to track down.